common/test/java/com/couchbase/lite/CollectionTest.java

```diff
@@ -0,0 +1,283 @@
+//
+// Copyright (c) 2022 Couchbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.couchbase.lite;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.junit.Test;
+
+import com.couchbase.lite.internal.utils.TestUtils;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+
+public class CollectionTest extends BaseCollectionTest {
+
+    //---------------------------------------------
+    //  Get Document
+    //---------------------------------------------
+    @Test
+    public void testGetNonExistingDocWithID() { assertNull(testCollection.getDocument("non-exist")); }
+
+    // get doc in the collection
+    @Test
+    public void testGetExistingDocInCollection() throws CouchbaseLiteException {
+        final String docId = "doc1";
+        createSingleDocInCollectionWithId(docId);
+        verifyGetDocumentInCollection(docId);
+    }
+
+    //---------------------------------------------
+    //  Save Document
+    //---------------------------------------------
+
+    // base test method
+    private void testSaveNewDocInCollectionWithID(String docID) throws CouchbaseLiteException {
```

Comment:
private methods go last in the class...

... also, by conventions, only methods that are @Test have signatures that begin with "test".  Maybe use the suffix "Test" as you did in the base class?